src/Exporter.php

```diff
@@ -69,7 +71,16 @@ public function shortenedRecursiveExport(array &$data, ?RecursionContext $proces
         /* @noinspection UnusedFunctionResultInspection */
         $processed->add($data);
 
+        $i     = 0;
+        $count = count($data, COUNT_RECURSIVE);
+
         foreach ($array as $key => $value) {
+            if ($count > self::MAX_SHORTENED_ITEMS && $i > self::MAX_SHORTENED_ITEMS) {
```

Comment:
since we are in the `shortenedRecursiveExport` method, I think its acceptable to shorten these arrays more agressively.

before this PR running phpunit on a dataprovider with lots of elements resulted in a big wall-of-text description

Reply:
Completely agree and thank you for this PR!